oak-run/src/main/java/org/apache/jackrabbit/oak/run/RevisionsCommand.java

```diff
@@ -82,6 +97,7 @@ public class RevisionsCommand implements Command {
             "  collect      perform garbage collection",
             "  reset        clear all persisted metadata",
             "  sweep        clean up uncommitted changes",
+            "  cleanup  clean up old/unused revisions",
```

Comment:
We might be able to handle this under the existing `collect` command - that one currently runs classic GC, but you can get it to do detailed GC as well, with the corresponding parameter.

(What's then missing is the ability to restrict it to specifically only a path)

Reply:
I agree we should at some point combine some of the commands into the same one.
For now, I have changed this to `pathCleanup`. But I think a better name can be found.